spec/parsePdr/ParsePdrSuccessSpec.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs');
+const workflow = require('@cumulus/integration-tests');
+const aws = require('@cumulus/common/aws');
+const { loadConfig, templateFile } = require('../helpers/testUtils');
+const awsConfig = loadConfig();
+const taskName = 'ParsePdr';
+
+const inputTemplateFilename = './spec/parsePdr/ParsePdr.input.template.json';
+const templatedInputFilename = templateFile({
+  inputTemplateFilename,
+  config: awsConfig[taskName]
+});
+const expectedParsePdrOutput = JSON.parse(fs.readFileSync('./spec/parsePdr/ParsePdr.output.json'));
+const pdrFilename = 'MOD09GQ_1granule_v3.PDR';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
```

Comment:
I think we can set this globally in `spec/run.js`. I should have moved it there.